invenio_app_rdm/upgrade_scripts/migrate_2_0_to_3_0.py

```diff
@@ -68,7 +68,7 @@ def execute_upgrade():
     # Cleanup associated deleted drafts.
 
     drafts = RDMDraftMetadata.query.filter(
-        RDMDraftMetadata.is_deleted == True
+        RDMDraftMetadata.is_deleted == True  # noqa
```

Comment:
the `noqa` prevents linters from complaining about this line or even attempting to fix it, because the `==` operator is overloaded by `SQLAlchemy`